### DISCUSSION TITLE
Flow Launcher plugin

### DISCUSSION THREAD

**Discussion by bretthysuik** [IgnisDa/ryot#341] [Show and tell]
Hi, I made a [Flow Launcher](https://www.flowlauncher.com/) plugin that lets you quickly search your ryot tracker:

![image](https://imgur.com/gVt9VBd.gif)

Repo: https://github.com/bretthysuik/Flow.Launcher.Plugin.Ryot

The next release of the plugin (v1.2.0) will support visual novels.

**Comment by IgnisDa** [comment]
Hey @bretthysuik, this looks neat! If you can, I would love if you could add this to the integration docs.

**Reply by bretthysuik** [reply]
Sure, I can do that
